app/src/main/java/org/schabi/newpipe/player/Player.java

```diff
@@ -748,10 +749,12 @@ private void initPlayback(@NonNull final PlayQueue queue,
                               final float playbackPitch,
                               final boolean playbackSkipSilence,
                               final boolean playOnReady,
-                              final boolean isMuted) {
+                              final boolean isMuted,
+                              final boolean shuffleEnabled) {
         destroyPlayer();
         initPlayer(playOnReady);
         setRepeatMode(repeatMode);
+        onShuffleModeEnabledChanged(shuffleEnabled);
```

Comment:
Just do `onShuffleModeEnabledChanged(false)` here and remove all of the other code changes, that would eventually be done in another PR saving the shuffle state

Reply:
yeah I did it in such a way that the intent could be changed to add the shuffle mode but if that isn't needed I'll switch to what you recommended